py-polars/polars/internals/expr.py

```diff
@@ -1632,7 +1632,7 @@ def apply(
 
         Depending on the context it has the following behavior:
 
-        ## Context
+        Context
```

Comment:
tbh maybe we just remove this line altogether. wdyt?

Reply:
👍